### ISSUE TITLE
Spinner does not show if disable=true

### ISSUE THREAD

**Issue by brycekmartin** [michaeldoye/mat-progress-buttons#31]
### Bug Report or Feature Request (mark with an `x`)
```
- [X ] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
Windows 7

### Versions
Angular 7

### Repro steps
Use the stackblitz demo to test what would happen if I change someFunc to look like this...
`  someFunc(): void {
    this.spinnerButtonOptions.active = true;
    this.spinnerButtonOptions.disabled = true;
    setTimeout(() => {
      this.spinnerButtonOptions.active = false;
      this.spinnerButtonOptions.disabled = false;
    }, 3500)
  }`

### The log given by the failure
The spinner did not show up, the button simply went disabled.

### Desired functionality
I would still want the spinner to show so that the user gets the feedback but want the button disabled so they can resubmit while the current request is processing.  I tested with the progress button and it works this way, but I like the look of the spinner better.

**Comment by brycekmartin** [comment]
Ohh geez, there I go again... upon looking at this further, it does disable the button while the spinner is active, it just doesn't change the button to the gray disabled color like the progress button.  Should these function in a consistent manner?

**Comment by michaeldoye** [comment]
Yeah, they should definitely be more consistent in behaviour. Thanks for reporting. I will make an effort to fix this for the next release.

**Comment by michaeldoye** [comment]
fixed in [8.0.4](https://github.com/michaeldoye/mat-progress-buttons/releases/tag/v8.0.4). Thanks!
